workflow-dev/tpl/objectstorage.toml

```diff
@@ -1,4 +1,4 @@
-#helm:generate helm template -o $HELM_GENERATE_DIR/manifests/deis-objectstorage-secret.yaml -d $HELM_GENERATE_FILE $HELM_GENERATE_DIR/tpl/deis-objectstorage-secret.yaml
```

Comment:
don't we still want to keep this? I still think we want to generate `manifests/deis-objectstorage-secret.yaml` when `helm generate` is called.